### PR TITLE
[AppService] az webapp auth update: Add optional parameter to update runtime-version

### PR DESCRIPTION
**Pull request by mkarmark**
**Description**  
We want to offer support via the Azure CLI tool to update the runtime version of the App Service Authentication/Authorization module. This is done by exposing a runtime version parameter in the "webapp auth update" command, already offered through the Azure CLI tool.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -724,7 +724,7 @@ def get_auth_settings(cmd, resource_group_name, name, slot=None):
 
 
 def update_auth_settings(cmd, resource_group_name, name, enabled=None, action=None,  # pylint: disable=unused-argument
-                         client_id=None, token_store_enabled=None,  # pylint: disable=unused-argument
+                         client_id=None, token_store_enabled=None, runtime_version=None,  # pylint: disable=unused-argument
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py:727]: will be param be passed to sdk call? didn't see it

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:727]: `runtime_version` is not used.

**Comment by qwordy** [comment]
The PR title does not conform to the requirement.
https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#format-pr-title

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py (hunk 1 above)
Comment (mkarmark) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:727]: runtime_version will be set in lines 752-4

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:727]: I am still confused. I search "runtime_version" and can't find usage.

**Comment by panchagnula** [comment]
@mkarmark can you add a test case for this - I am still confused how this is working? You might also need to update params.py and help for this new property.

**Comment by mkarmark** [comment]
@panchagnula Added the test in the latest PR, but a lot of the work (i.e. params.py) for runtime version was already done, I am just connecting the final pieces in order to get cli support for pinning.

**Comment by yonzhan** [comment]
add to S171

**Comment by mkarmark** [comment]
@panchagnula, here is the expected behavior demonstrated with the runtime version listed in an invalid format
image

**Comment by panchagnula** [comment]
> @panchagnula, here is the expected behavior demonstrated with the runtime version listed in an invalid format
> image

this means if we pass even an invalid format the API just sets the value? is this API going to handle this? if not should CLI validate the string & then pass it to API?

**Comment by mkarmark** [comment]
> > @panchagnula, here is the expected behavior demonstrated with the runtime version listed in an invalid format
> > image
> 
> this means if we pass even an invalid format the API just sets the value? is this API going to handle this? if not should CLI validate the string & then pass it to API?

I pushed some changes that result in the following error message upon an invalid version:
image

**Pull request merged**